e2e/tests/workspace/old-cdr-version-modal.spec.ts

```diff
@@ -50,4 +50,29 @@ describe('OldCdrVersion Modal restrictions', () => {
     const finishButton = workspaceEditPage.getDuplicateWorkspaceButton();
     expect(await finishButton.isCursorNotAllowed()).toBe(true);
   });
+
+  test('OWNER cannot bypass older CDR Version restrictions by clicking cancel', async () => {
+    const workspaceCard = await findOrCreateWorkspaceCard(page, { workspaceName: workspace });
+    await workspaceCard.asElementHandle().hover();
+    await workspaceCard.selectSnowmanMenu(MenuOption.Duplicate, { waitForNav: true });
+
+    const workspaceEditPage = new WorkspaceEditPage(page);
+
+    // fill out the fields required for duplication and observe that duplication is enabled
+    await workspaceEditPage.fillOutRequiredDuplicationFields();
+    const duplicateButton = workspaceEditPage.getDuplicateWorkspaceButton();
+    await duplicateButton.waitUntilEnabled();
```

Comment:
nit: Missing a assertion to be sure. `expect(await duplicateButton.isCursorNotAllowed()).toBe(false);`

Reply:
I saw this pattern in various places and I was confused by it.  Why is it necessary to also check the cursor if we're already waiting for the button to be enabled?

Reply:
What you saw was because `waitUntilEnabled` didn't throw out exception at the beginning. So the test continued even after error. 
Assertion is not necessary. Test should stops now when `waitUntilEnabled` fails. I like to see assertions because it make reading steps easier.